src/views/AmalgamationSelection.vue

```diff
@@ -220,17 +222,25 @@ export default class AmalgamationSelection extends Vue {
    * @returns the business identifier of the newly created amalgamation application
    */
   private async createBusinessAA (type: AmalgamationTypes): Promise<string> {
-    const accountId = +JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.id || 0
-    const email = this.isShortFormAmalgamation(type) ? this.getBusinessEmail : ''
-    const phone = this.isShortFormAmalgamation(type) ? this.getFullPhoneNumber : ''
-    const legalName = this.isShortFormAmalgamation(type) ? this.getLegalName : ''
-    const correctNameOption = this.isShortFormAmalgamation(type) ? CorrectNameOptions.CORRECT_AML_ADOPT : null
+    // short-form amalgamations use the legal name of the primary business,
+    // otherwise don't set a legal name
+    const legalName = this.isShortFormAmalgamation(type) ? this.getLegalName : undefined
```

Comment:
Setting an object property to `undefined` is almost like removing it. Certainly, when the network code converts the object to JSON, the property is not saved.